packages/integration-tests/projects/suite-web/support/pageObjects/accountsObject.ts

```diff
@@ -0,0 +1,16 @@
+/// <reference types="cypress" />
+
+class AccountsPage {
+    unpackAllAccountTypes() {
+        cy.getTestElement('@account-menu/arrow').click({ multiple: true });
+    }
+
+    applyCoinFilter(coin: string) {
```

Comment:
nitpick: I would consider making `coin` arg  strongly typed based on https://github.com/trezor/trezor-suite/blob/develop/suite-common/wallet-config/src/networksConfig.ts#L386
feel free to ignore